EVENTS.md

```diff
@@ -9,67 +9,57 @@ key.
+|                      | name      | string   | `Agility`                  | Name                                                   |
+|                      | value     | float    | `50.8`                     | Points value                                           |
+|                      | type      | string   | `skill`                    | Type of point `skill` or `attribute`                   |
+| **damage_inflicted** |           |          |                            | Player inflicted damage                                |
+|                      | amount    | float    | `50.0`                     | Amount of damage inflicted                             |
+|                      | critical  | bool     | `true`                     | Critical hit                                           |
+| **damage_taken**     |           |          |                            | Player took damage                                     |
+|                      | amount    | float    | `50.0`                     | Amount of damage taken                                 |
+|                      | critical  | bool     | `false`                    | Critical hit                                           |
+| **enemy_evade**      |           |          |                            | Enemy evaded damage                                    |
+|                      | reason    | string   | `dodge`                    | Possible values: `dodge`, `evade`, `jam` or `miss`     |
+| **enhancer_break**   |           |          |                            | Enhancer broke                                         |
+|                      | name      | string   | `Weapon Damage Enhancer 1` | Name of the enhancer                                   |
+|                      | item      | string   | `Omegaton M83 Predator`    | Item it was attached to                                |
+|                      | remaining | int      | `200`                      | Currently remaining on item                            |
+|                      | value     | float    | `0.8000`                   | PED value                                              |
+| **special_loot**     |           |          |                            | Player got a special loot**                            |
+|                      | value     | int      | `15`                       | PED value                                              |
+|                      | player    | string   | `Some Player Name`         | Name of the player                                     |
+|                      | enemy     | string?  | `Kerberos Young`           | Enemy name                                             |
+|                      | item      | string?  | `Holy Grail`               | Only available on `rare` loots                         |
+|                      | type      | string   | `global`                   | Possible values: `global`, `hall_of_fame` or `rare`    |
```

Comment:
I think some hof/global messages had a location in land areas and on certain planets. We should probably add that here too.

Reply:
Makes sense